testsuite/simulation/modelica/solver/problem6-cvode.mos

```diff
@@ -83,7 +83,7 @@ if res2[1,s] > -1e-2 then 1 else 0;
 // true
 // true
 // 1.0
-// 1.0
+// 0.0
```

Comment:
I guess this was wrong the whole time and it is now correct. That is a good sign.

Reply:
Yes. I'm not sure why I didn't noticed at first, but now it should be correct.